### DISCUSSION TITLE
Cann't login, "Unable to detect login share from the Auth Network. This may be due to slow internet connection. Check your internet speed and try again. If you're using a vpn, please turn it off."

### DISCUSSION THREAD

**Discussion by Douglasacost** [Web3Auth/Web3Auth#1228] [Help]
I'm trying to implement web3Auth, but since a week can not loggin 

I've set up a "Plug and play" project pointing to "Cyan" network. 
On trying it the popup shows "Constructing key from: 127.0.0.1" then shows "Unable to detect login share from the Auth Network. This may be due to slow internet connection. Check your internet speed and try again. If you're using a vpn, please turn it off."

I've checked to be using the correct Client ID 

Thanks for help

**Comment by VoR0220** [comment]
Also running into this on a React Native App on testnet. Hoping this is a temporary thing because I have very high speed internet and checked with adblocker and VPN off, still failing.

**Comment by arch1995** [comment]
hey @Douglasacost @VoR0220 

Please share the following details:
- SDK(s) and its version
- Web3auth initialization & login code snippets.
- Browser console.
- Screenshots (if any)

**Reply by Douglasacost** [reply]
It was an bad parameter on the configuration. But the error es misleading tho. As a dev at first I thought the error was pointing to your side
"Unable to detect login share from the Auth Network. This may be due to slow internet connection. Check your internet speed and try again. If you're using a vpn, please turn it off."

**Reply by VoR0220** [reply]
I also got around this seemingly without any interaction. But this maybe needs to be a bit better documented in the errors.

**Reply by pmn4** [reply]
@Douglasacost what was the configuration error? I am trying to set up for the first time and am getting the same error

**Reply by Douglasacost** [reply]
On my side it was the network. I set 'cyan' instad of 'testnet' while in the dashboard all was pointing to 'testnet'
Also you will required an extra eye if you setup a custom verifier
